data_structures/bloom_filters/bloom_filter/bloom_filter.c

```diff
@@ -0,0 +1,146 @@
+#include "./bloom_filter.h"
+
```

Comment:
That ./ probably isn't necessary, #include "bloom_filter.h" should do it. Also I believe extended integer types requires stdint.h or inttype.h
```suggestion
#include "bloom_filter.h"
#include <stdint.h>
```